components/Elevation/src/MDCElevatable.h

```diff
@@ -18,7 +18,7 @@
 /**
  Provides APIs for @c UIViews to communicate their elevation throughout the view hierarchy.
  */
-@protocol MDCElevatable
```

Comment:
The alternative is the blocks themselves expect an `NSObject` rather than an `id`. It seems that the `NSObject` is required for the block and not the protocol - what do you think? Is there a specific reason to put the change on the protocol rather than the blocks?

Reply:
I don't have a preference with either approach, let me know if you do and I'll update accordingly.

Reply:
Changing the block to be NSObject<> caused more propagated changes across UIView+MaterialElevationResponding. I'm going down the protocol update route. Please feel free to open a follow up bug/PR if you find issues with this